package.json

```diff
@@ -2,18 +2,23 @@
   "name": "min-dom",
   "version": "4.1.0",
   "description": "A minimal dom utility toolbelt",
-  "main": "dist/index.js",
-  "module": "dist/index.esm.js",
-  "types": "dist/index.d.ts",
+  "type": "module",
+  "exports": {
+    ".": {
+      "import": "./dist/index.js",
+      "types": "./dist/index.d.ts"
```

Comment:
Is this how to publish types with an ES module? I couldn't find [any information](https://www.typescriptlang.org/docs/handbook/declaration-files/publishing.html). 🤔

Reply:
Evidence shows that we need to do two things:

* Publish types via a `types` property
* (Optional) add to `exports`, too, though typescript@5.3 also discovers them without it.

Cf. [min-dash](https://github.com/bpmn-io/min-dash/blob/main/package.json), which is a blueprint for this an other changes.

Reply:
But also [lezer-feel](https://github.com/nikku/lezer-feel/blob/main/package.json) where we use the legacy exports no more, but only `exports` field.

`main` has week semantics in `type: module` packages, hence this move is encouraged.

Reply:
> * Publish types via a types property

So should we add `types` as well?